setup.py

```diff
@@ -42,6 +43,8 @@ def find_stub_files(name):
     url="https://github.com/typeddjango/djangorestframework-stubs",
     author="Maksim Kurnikov",
     author_email="maxim.kurnikov@gmail.com",
+    maintainer="Marti Raudsepp",
+    maintainer_email="marti@juffo.org",
```

Comment:
@sobolevn I noticed that maintainer was unset in djangorestframework-stubs `setup.py`. Should I update the maintainer field in both projects to myself?

OTOH if we ever go to a shared maintainership model, maybe it makes sense to remove it from both projects.

Reply:
Please, feel free :)

It does not affect anything, but it is a nice way to say "thank you"!